__tests__/app.test.js

```diff
@@ -80,6 +80,14 @@ describe("/api/articles/:article_id", () => {
         expect(msg).toBe("Not found");
       });
   });
+  test("should respond with an article object including comment_count property", () => {
+    return request(app)
+      .get("/api/articles/9")
+      .expect(200)
+      .then(({ body: { article } }) => {
+        expect(article).toHaveProperty("comment_count", 2);
```

Comment:
I think having an assertion with `.toMatchObject` and checking all the properties including the comment_count would be a more robust test.